### DISCUSSION TITLE
Ignoring other formatters with eslint-plugin-prettier recipe

### DISCUSSION THREAD

**Discussion by cswingle** [LazyVim/LazyVim#604] [Q&A] [answered]
I'm using the `volar` and `eslint` language servers for Vue files and my project uses the `eslint-plugin-prettier`, so I added the recipe [Add Eslint and use it for formatting](https://www.lazyvim.org/configuration/recipes#add-eslint-and-use-it-for-formatting). The problem is that both prettier/eslint *and* Volar are formatting the document on save and I only want prettier/eslint to format it or Vue complains about the formatting. In the rare case that Volar times out, I get the formatting I want, so I think the recipe is working, but I don't know how to tell it not to use Volar for formatting. I have made some attempts at adding eslint/prettier to `null-ls` so that the formatting will ignore the language servers for formatting (something the recipe specifically says not to do...) but I haven't managed to figure out how to get it to work. Thanks!

**Comment by folke** [comment, marked as answer]
You'll have to disable formatting for Volar. Check their lsp docs

**Reply by cswingle** [reply]
Thanks, that makes sense.
